docs/usage/tcja_after_2025.md

```diff
@@ -0,0 +1,133 @@
+Provisions in the "Tax Cuts and Jobs Act" (TCJA, P.L. 115-97)](
+https://crsreports.congress.gov/product/pdf/R/R47846), which is dated
+November 21, 2023.
+
+This document describes how to use the PSLmodels Tax-Calculator
+command-line interface
+[tc](https://taxcalc.pslmodels.org/guide/cli.html) with any compatible
+input dataset to analyze the post-2025 effects of alternatives to
+current-law policy (which calls for all temporary TCJA provisions to
+expire).  Compatible input datasets include:
+
+* the older `cps.csv` file included in the Tax-Calculator package
+
+* the older `puf.csv` file generated in the PSLmodels
+  [taxdata](https://github.com/PSLmodels/taxdata) repository and
+  available only to those with access to the 2011 IRS/SOI PUF
+
+* several newer CSV-formatted input files created in the PSLmodels
+  [tax-microdata](https://github.com/PSLmodels/tax-microdata-benchmarking)
+  repository that are based on the 2015 IRS/SOI PUF and on recent CPS
+  data, and are available only to those with access to the 2015
+  IRS/SOI PUF
+
+Before reading the rest of this document, be sure you understand how
+to use the Tax-Calculator command-line tool
+[tc](https://taxcalc.pslmodels.org/guide/cli.html), particularly the
+`--baseline` and `--reform` command-line options.  (For complete and
+up-to-date `tc` documentation, enter `tc --help` at the command
+prompt.)  Omitting the `--baseline` option means the baseline policy
+is current-law policy.  Omitting the `--reform` option means the
+reform policy is current-law policy.  The `--tables` option produces
+tables comparing taxes under the baseline policy to taxes under the
+reform policy by income decile and in aggregate.
+
+The `--baseline` option is not commonly used, but it can be very
+helpful in analyzing reforms that take effect beginning in 2026.  Such
+a reform could be analyzed relative to current-law policy (that is,
+with temporary TCJA provisions expiring after 2025) by omitting the
+`--baseline` option.  But if such a reform were to be analyzed
+relative to extending all the temporary TCJA provisions beyond 2025,
+then the `--baseline ext.json` option would need to be used.  The
+[`ext.json`](./ext.json) file contains the 2026 tax policy reform
+provisions that would extend TCJA's temporary provisions beyond 2025.
+Before using this `ext.json` reform file, be sure to read how it is
+generated at the end of this document.
+
+Here are some concrete examples of using the `tc` tool to analyze a
+reform of interest to you that begins in 2026.  The examples assume
+you have named your reform file `x.json` and that you are using one
+of the compatible input datasets describe above.  The examples will
+call the input dataset `z.csv`.
+
+To analyze your reform relative to current-law policy (which means
+temporary TCJA provisions have expired beginning in 2026), you would
+execute this command:
+
+```
+tc z.csv 2026 --exact --tables --reform x.json
```

Comment:
This instructions are very helpful, but to ease the burden on those replicating them, and to provide a concrete example, should their be a full example in the "recipes" using the current policy baseline?

Reply:
@jdebacker said:
> This instructions are very helpful, but to ease the burden on those replicating them, and to provide a concrete example, should their be a full example in the "recipes" using the current policy baseline?

I don't have the time to do that and I don't have any experience with Jupyter files, which is what the Recipes have been converted to.   If you want a recipe, ask someone with the necessary skills.